modules/navier_stokes/src/auxkernels/INSFVSlipVelocityAux.C

```diff
@@ -0,0 +1,143 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "INSFVSlipVelocityAux.h"
+#include "INSFVVelocityVariable.h"
+#include "Function.h"
+#include "NS.h"
+
+registerMooseObject("NavierStokesApp", INSFVSlipVelocityAux);
+
+InputParameters
+INSFVSlipVelocityAux::validParams()
+{
+  InputParameters params = AuxKernel::validParams();
+  params.addClassDescription("Computes the turbulent viscosity for the mixing length model.");
+  params.addRequiredCoupledVar("u", "The velocity in the x direction.");
+  params.addCoupledVar("v", "The velocity in the y direction.");
+  params.addCoupledVar("w", "The velocity in the z direction.");
+  params.addRequiredParam<MooseFunctorName>(NS::density, "Continuous phase density.");
+  params.addRequiredParam<MooseFunctorName>("rho_d", "Dispersed phase density.");
+  params.addRequiredParam<MooseFunctorName>(NS::mu, "Mixture Density");
+  params.addParam<RealVectorValue>(
+      "gravity", RealVectorValue(0, 0, 0), "Gravity acceleration vector");
```

Comment:
```suggestion
      "gravity", "Gravity acceleration vector");
```
and make it required

Reply:
Thanks. Changed

Reply:
I didn't make it required because I want to add a default value of '0.0 0.0 0.0' for no gravitational acceleration